src/structures/party/ClientPartyMember.ts

```diff
@@ -213,7 +253,10 @@ class ClientPartyMember extends PartyMember {
       ...data,
       AthenaCosmeticLoadout: {
         ...data.AthenaCosmeticLoadout,
-        characterDef: `/Game/Athena/Items/Cosmetics/Characters/${cid}.${cid}`,
+        characterDef: `${
+          path?.replace(/\/$/, '') ??
+          '/BRCosmetics/Athena/Items/Cosmetics/Characters'
```

Comment:
As of right now, there are some cosmetics that still have the old path, example: Renegade Raider ([fortnite-api.com request for the cosmetic](https://fortnite-api.com/v2/cosmetics/br/CID_028_Athena_Commando_F)) 

I also checked with FModel just to be sure and the api's correct 
![image](https://github.com/fnbrjs/fnbr.js/assets/64664639/120ea212-dae4-49e1-89dc-cb6932f032bf)

What are you gonna do about those cosmetics? Should we just shrug it off and wait for a Fortnite update?

Reply:
you can specify the path manually in the last param, and you can check if the cosmetic uses the new or the old path with fortnite-api.com ![image-45.png](https://github.com/fnbrjs/fnbr.js/assets/57068341/fb7d487a-076e-45d6-82f1-f667c71c0c99)

 ![image-37.png](https://github.com/fnbrjs/fnbr.js/assets/57068341/e683d06e-abfd-44bf-9b24-70088640f275)

Reply:
oh okay, i never noticed that param when using the lib